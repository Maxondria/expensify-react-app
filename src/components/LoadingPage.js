import React from "react";
import loader from "../images/loader.gif";

export default () => (
  <div className="loader">
    <img className="loader__image" src={loader} alt="Loading..." />
  </div>
);
