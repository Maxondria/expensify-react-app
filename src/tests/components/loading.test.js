import React from "react";
import LoadingPage from "../../components/LoadingPage";
import { shallow } from "enzyme";

it("should render the loading page correctly", function() {
  const wrapper = shallow(<LoadingPage />);
  expect(wrapper).toMatchSnapshot();
});
