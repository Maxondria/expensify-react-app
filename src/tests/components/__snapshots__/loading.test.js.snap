// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the loading page correctly 1`] = `
<div
  className="loader"
>
  <img
    alt="Loading..."
    className="loader__image"
    src="loader.gif"
  />
</div>
`;
